code/_onclick/hud/human.dm

```diff
@@ -311,6 +311,10 @@
 	internals.hud = src
 	infodisplay += internals
 
+	spacesuit = new /atom/movable/screen/spacesuit
+	spacesuit.hud = src
+	infodisplay += spacesuit
```

Comment:
Does this show at all times?

Reply:
https://github.com/user-attachments/assets/6ed9cb56-3424-4a3b-b906-880e32b20ef3

Yes you are right. It was not there because I didnt implement it correctly.

Here it is when you equip a hardsuit